docs/data-sources/ec_deployment.md

```diff
@@ -32,6 +32,7 @@ These will not be available for interpolation.
 * `deployment_template_id` - ID of the deployment template used to create the deployment.
 * `traffic_filter` - Traffic filter block, which contains a list of traffic filter rule identifiers.
 * `observability` Observability settings. Information about logs and metrics shipped to a dedicated deployment.
+* `tags` Metadata tags. Arbitrary tags applied to the deployment.
```

Comment:
Let's move this line above the `observability` line because the lines below are part of the observability object. Also we should probably change the text to say what it says on the resource [description](https://github.com/elastic/terraform-provider-ec/blob/master/docs/resources/ec_deployment.md#argument-reference).
```suggestion
* `tags` Key value map of arbitrary string tags..
```